docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,232 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, pipelining aims to increase the
+throughput of the entire network.
+
+### Synchronous vs. Asynchronous Backing
 
 Asynchronous Backing has three overarching goals:
 
 1. Decrease parachain block (parablock) validation time to **6 seconds** from **12 seconds**
-2. Increase the amount of usable blockspace, allowing more state changes per relay chain block.
-3. Allow for parablocks to be re-proposed to the network if they are not included successfully on
+2. **Increase the usable blockspace**, allowing more state changes per relay chain block.
+3. **Allow parablocks to be re-proposed** to the network if they are not included successfully on
    the first attempt.
 
-Asynchronous backing provides a form of **contextual execution**, which allows for more time for
-parachain collators to fit more transactions and prepare block candidates for backing and inclusion.
-**Contextual execution** refers to how a parablock can be built earlier using the context provided
-by an _unincluded segment_ of recent block ancestors.
+In synchronous backing, parablock generation relies on the most recent relay-chain block. Each
+parablock must be generated and go through the entire backing process in a single relay block. It
+then proceeds to the inclusion process while generating the next relay-chain block.
+
+Asynchronous backing brings the following changes to the parachain protocol:
 
-**Unincluded segments** are chains of candidate blocks that are not yet included in the relay chain.
-Parablocks can be added to this unincluded segment without waiting for the latest included parent
-block of the relay chain. The core functionality that asynchronous backing brings is the ability to
-build on these unincluded segments of block ancestors rather than ancestors included in the relay
-chain state.
+- Parablock generation is now decoupled from the backing process, avoiding the previous 6-second
+  deadline through the backing process. This enables for more involved scaling approaches, such as
+  giving more time for collators to properly use blockspace or increasing block velocity based on
+  demand.
 
-Currently, parablocks rely on the most recent relay chain block (often referred to as the **parent**
-block, as the parablock anchors itself to it). Each parablock must be generated and go through the
-entire backing process in a single relay block. It then proceeds to the availability + inclusion
-process during the next block.
+- Parachains can generate blocks and have them placed into **unincluded segments** of parablock
+  ancestors (i.e. parablocks that are seconded but not yet backed by paravalidators and included in
+  the relay chain) rather than ancestors included in the relay chain state.
 
-:::info
+### Unincluded Segments
 
-It is important to note the following clarifications:
+Unincluded segments are chains of candidate parablocks that are seconded but still need to be backed
```

Comment:
Unincluded segments can contain blocks that have been backed on the relay chain. They can contain parablocks at any stage pre-inclusion.
```suggestion
Unincluded segments are chains of candidate parablocks have yet to be included in the relay chain.
```